lists/opengl.md

```diff
@@ -3,6 +3,7 @@
 ### Types of content:
 - Articles/Blogs
 - Papers/Books
+- References
```

Comment:
change Papers/Books to Papers/Books/References and change the link below to reflect it